lib/evss/pciu_address/service.rb

```diff
@@ -48,15 +48,23 @@ def with_exception_handling
         log_message_to_sentry(e.message, :error, extra_context: { url: config.base_path })
         raise Common::Exceptions::Forbidden, detail: 'Missing correlation id'
       rescue Common::Client::Errors::ClientError => e
-        raise Common::Exceptions::Forbidden if e.status == 403
-        log_message_to_sentry(
-          e.message, :error, extra_context: { url: config.base_path, body: e.body }
-        )
+        log_message_to_sentry(e.message, :error, extra_context: { url: config.base_path, body: e&.body })
```

Comment:
I typically log the exception when there is one, (e.g. `log_exception_to_sentry`), however, it's also fine to use `log_message_to_sentry` if you think it will be more meaningful for someone viewing Sentry.